src/docs/background/02-concepts/index.md

```diff
@@ -56,26 +49,33 @@ See the
 [Getting started with Flex CLI](/tutorials/getting-started-with-flex-cli/)
 tutorial for more information.
 
-### Integration
-
-An external application that communicates with the marketplace. This can
-mean reacting to marketplace events, pulling data from the marketplace,
-or triggering activity on the marketplace. For example, “automatically
-update a Mailchimp email list when a new user joins the marketplace”.
-
 ### Marketplace API
 
-The application programming interface that provides the crucial features
-marketplaces need. For example, the Marketplace API allows for creating
-and searching content (like user profiles and listings), managing user
-accounts, and processing transactions.
+The application programming interface (API) that provides the crucial features
+marketplaces need. For example, the Marketplace API allows for creating and
+searching content (like user profiles and listings), managing user accounts, and
+processing transactions.
 
 Sharetribe customers can create their own web-based and mobile UIs on
 top of the Marketplace API.
 
 Information about working with the Marketplace API can be found in the
 [API reference documentation](/references/api/).
 
+### Integration API
+
+The application programming interface (API) that provides full access to
```

Comment:
I'd stick with "The" here, because "The Integration API" is one concrete API we are talking about. Same in your suggestion above.

Reply:
Interesting! I'd select "a" for quite a similar reason but use the following sentence as an example:
_The Integration API is an application programming interface that provides..._

Reply:
Yeah, in the next sentence "an" is better, because we mention the concrete thing first and then explain what it is. But when introducing it first, I'd still use "the".

Reply:
I can see how "The" works there, but would still argue that an "An" suits here too. Let's keep the "The" and continue the discussion IRL and we can update this if it feels necessary.

Reply:
It does fit too, yes.